doc/releases/changelog-dev.md

```diff
@@ -4,19 +4,78 @@
 
 <h3>New features since last release</h3>
 
-* A thermal relaxation channel is added to the Noisy channels. The channel description can be 
+* A thermal relaxation channel is added to the Noisy channels. The channel description can be
   found on the supplementary information of [Quantum classifier with tailored quantum kernels](https://arxiv.org/abs/1909.02611).
   [(#1766)](https://github.com/PennyLaneAI/pennylane/pull/1766)
-  
-* Added the identity observable to be an operator. Now we can explicitly call the identity 
+
+* Added the identity observable to be an operator. Now we can explicitly call the identity
   operation on our quantum circuits for both qubit and CV devices.
-  [(#1829)](https://github.com/PennyLaneAI/pennylane/pull/1829) 
+  [(#1829)](https://github.com/PennyLaneAI/pennylane/pull/1829)
+
+* A function for computing generalized parameter shift rules for generators'
+  whose eigenvalue frequency spectrum is known is available as `qml.gradients.get_shift_rule`.
+  Given a generator's frequency spectrum of `R` unique frequencies, `qml.gradients.get_shift_rule`
```

Comment:
```suggestion
* A function for computing generalized parameter shift rules for generators'
  whose eigenvalue frequency spectrum is known is available as `qml.gradients.get_shift_rule`.
  [(#1788)](https://github.com/PennyLaneAI/pennylane/pull/1788)

  Given a generator's frequency spectrum of `R` unique frequencies, `qml.gradients.get_shift_rule`
```
may have accidentally gotten deleted in the last suggestion!

Reply:
This sentence is kinda hard to parse. I'd reword as
"For generators' whose eigenvalue frequency spectrum is known, `qml.gradients.get_shift_rule` is a function that computes the generalized parameter shift rules."
Or something similar

Reply:
or maybe even?

> `qml.gradients.get_shift_rule` computes the generalized parameter-shift rule of arbitrary operations, given the operations' generator spectrum.

I realize the current sentence is a bit ambiguous when referring to generators

Reply:
In the latest commit I've changed to: 

"For Hamiltonians whose eigenvalue frequency spectrum is known, `qml.gradients.get_shift_rule` is
  a function that computes the generalized parameter shift rules for the time evolution."

I'd say this still emphasizes conciseness over concreteness, but at least avoids introducing "generator" terminology.